requestor/run_service_manager.py

```diff
@@ -0,0 +1,44 @@
+import asyncio
+
+from service_manager import ServiceManager
+import erigon_services
+
+
+def state(service_wrapper):
+    if not service_wrapper.started:
+        status = 'pending'
```

Comment:
I think function with multiple `return`s is less readable (not very important in this case).

Also it is harder to add a poor man debug like `print("COMPUTED STATE: ", state)`.
Also it requires more work to change the interface e.g. to
```
return {'status': status, 'something_else': something_else}
```

Those are my reasons, I'm not saying they are good reasons : )

Reply:
>I think function with multiple `return`s is less readable (not very important in this case).

Totally don't share this opinion but I won't fight over it. Your reasons are valid, it can stay like this.